networkit/cpp/distance/DynBFS.cpp

```diff
@@ -34,6 +34,59 @@ void DynBFS::run() {
             maxDistance = distances[v];
     });
     maxDistance++;
+    hasRun = true;
+} */
+
+void DynBFS::run() {
```

Comment:
The old DynBFS::run() is commented out in this PR and replaced by a new function. Is this change intended? Why is it necessary?